subsys/bluetooth/audio/shell/bap.c

```diff
@@ -3412,7 +3474,7 @@ static int cmd_create_broadcast_sink(const struct shell *sh, size_t argc, char *
 			.options = BT_LE_SCAN_OPT_NONE,
 			.interval = BT_GAP_SCAN_FAST_INTERVAL,
 			.window = BT_GAP_SCAN_FAST_WINDOW,
-			.timeout = 1000, /* 10ms units -> 10 second timeout */
+			.timeout = 0,
```

Comment:
Alternatively, since we are using extended advertising and the data shall be in the advertising data and not scan response data, we could omit the changes to the timeout here and just using passive scan

Reply:
Ok, I’ll change it since your suggested alternative is simpler!